src/main/scala/com/gd/twitteranalytics/StreamingTweetsJob.scala

```diff
@@ -0,0 +1,51 @@
+package com.gd.twitteranalytics
+
+import com.gd.twitteranalytics.TweetsTransformer._
+import com.gd.twitteranalytics.TweetsDataLoader._
+import org.apache.log4j.Logger
+import org.apache.spark.SparkConf
+import org.apache.spark.streaming.dstream.DStream
+import org.apache.spark.streaming.{Seconds, StreamingContext}
+import java.util.{Date => JavaDate}
+import com.gd.twitteranalytics.util.AppConfigReader
+
+object StreamingTweetsJob {
+
+  val log = Logger.getLogger(StreamingTweetsJob.getClass.getName)
+
+  def main(args: Array[String]): Unit = {
+      val authKeys = ConfigValidator.getTwitterAccessKeys
```

Comment:
a validator is responsible for getting keys...
doesn't it feel right to you?
Could you elaborate please, what's the design intention behind of this? how does it comply with high cohesion concept?